tests/sock_udp_aux/main.c

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2020 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for PTP timestamping through the sock API
+ *
+ * @author      Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "fmt.h"
+#include "net/sock/udp.h"
+#include "shell.h"
+#include "shell_commands.h"
+#include "net/ipv6/addr.h"
+
+#define MAIN_QUEUE_SIZE     (8)
+
+#define PORT                12345
```

Comment:
```suggestion
#define PORT_UDP            12345
```

to avoid conflicts with vendor files